### PR TITLE
a bug fix in `flowwrite` and a `dist_train.sh` script

### PR DESCRIPTION
**Pull request by wwhio**

### CODE REVIEW

**Comment by xinntao** [comment]
Hi, @wwhio 

Thanks for your contribution, I will have a look now~ 
Sorry for the delay~

File: docs/TrainTest.md [hunk 1]
```diff
@@ -37,12 +37,24 @@ In general, both the training and testing include the following steps:
 > CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 \\\
 > python -m torch.distributed.launch --nproc_per_node=8 --master_port=4321 basicsr/train.py -opt options/train/EDVR/train_EDVR_M_x4_SR_REDS_woTSA.yml --launcher pytorch
 
+or
+
+> PYTHONPATH="./:${PYTHONPATH}" \\\
```
Comment (xinntao) [inline docs/TrainTest.md:42]: As `PYTHONPATH` has already been declared in dist_train.sh, remove it in the cmd

File: docs/TrainTest.md [hunk 2]
```diff
@@ -37,12 +37,24 @@ In general, both the training and testing include the following steps:
...
+
 **4 GPUs**
 
 > PYTHONPATH="./:${PYTHONPATH}" \\\
 > CUDA_VISIBLE_DEVICES=0,1,2,3 \\\
 > python -m torch.distributed.launch --nproc_per_node=4 --master_port=4321 basicsr/train.py -opt options/train/EDVR/train_EDVR_M_x4_SR_REDS_woTSA.yml --launcher pytorch
 
+or
+
+> PYTHONPATH="./:${PYTHONPATH}" \\\
```
Comment (xinntao) [inline docs/TrainTest.md:54]: remove PYTHONPATH="./:${PYTHONPATH}"

File: docs/TrainTest_CN.md [hunk 3]
```diff
@@ -37,12 +37,24 @@
 > CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 \\\
 > python -m torch.distributed.launch --nproc_per_node=8 --master_port=4321 basicsr/train.py -opt options/train/EDVR/train_EDVR_M_x4_SR_REDS_woTSA.yml --launcher pytorch
 
+或者
+
+> PYTHONPATH="./:${PYTHONPATH}" \\\
```
Comment (xinntao) [inline docs/TrainTest_CN.md:42]: remove PYTHONPATH="./:${PYTHONPATH}"

File: docs/TrainTest_CN.md [hunk 4]
```diff
@@ -37,12 +37,24 @@
...
+
 **4 GPUs**
 
 > PYTHONPATH="./:${PYTHONPATH}" \\\
 > CUDA_VISIBLE_DEVICES=0,1,2,3 \\\
 > python -m torch.distributed.launch --nproc_per_node=4 --master_port=4321 basicsr/train.py -opt options/train/EDVR/train_EDVR_M_x4_SR_REDS_woTSA.yml --launcher pytorch
 
+或者
+
+> PYTHONPATH="./:${PYTHONPATH}" \\\
```
Comment (xinntao) [inline docs/TrainTest_CN.md:54]: remove PYTHONPATH="./:${PYTHONPATH}"

**Comment by xinntao** [comment]
@wwhio  there are some comments, could you please fix them? 
Thanks !

**Comment by wwhio** [comment]
@xinntao Done. And I also add the `dist_test.sh` script.

**Pull request merged**